ETC_en.tsv

```diff
@@ -4234,7 +4234,7 @@ ETC_20150317_004233	{@st41} Gesture List
 ETC_20150317_004234	[Main]
 ETC_20150317_004235	{@st55_b}{b} Reward(s) {/}
 ETC_20150317_004236	{@st55_c} Experience :
-ETC_20150317_004237	Waiting for the other to accept.
+ETC_20150317_004237	Waiting for the the player to accept.
```

Comment:
Replace the second 'the' with 'other'. 
So: Waiting for the other player to accept.